### ISSUE TITLE
Using Mapstyle "Elevate4.xml for OpenAndroMap causes map to blank

### ISSUE THREAD

**Issue by Nahannidog** [ge0rg/aprsdroid#177]
Hello,
Installed 1.3.0-106-g2377223 2017-05-12, pointed to the OpenAndroMap (USA Utah & Colorado) [http://download.openandromaps.org/maps/usa/Utah_Colorado.zip](url).  Phone is Samsung Note3 at Android 5.0.  The map loads OK, but scaling and labeling is poor.  I then noticed that APRSdroid now can use Mapsforge Rendering themes.  I pointed APRSDroid to OpenAndroMaps recommended Elevate4.xml  (this combo looks great in Oruxmaps). [http://www.openandromaps.org/en/legend/elevate-mountain-hike-theme](url)

 After doing that, theAPRSDroid map is **blank** (I see the received station icons on blank map).  

My map is: storage/emulated/0/aprsdroid.map (renamed from Utah_Colorado.map)
My render them is:  storage/emulated/0/Elevate4/Elevate4.xml

I also tried the secondarily recommended Elements.xml with the same blank results

I see similar closed issues #86, #101 and #140 but I can't resolve.  I used ES File Explorer to choose the map and the mapstyle.

Thanks for your help!

**Comment by stephensworld** [comment]
Hi,

I have the same problem.

73
Stephan

**Comment by ge0rg** [comment]
APRSdroid is using mapsforge version 0.3, so you will need to test the Elevate2 style, not the Elevate4 map. Unfortunately, there is very little documentation about the theme files and how they work, so I'm flying blind here as well :)

**Comment by Nahannidog** [comment]
HI Georg,
Thanks, That's the solution!  I tried the suite of mapthemes at Elevate2  [http://www.openandromaps.org/en/legend/elevate-mountain-hike-theme#toggle-id-1](url) both the 320dpi style "L" and 480 dpi style "XL", all the themes (Backcountry, Hiking, Cycling, City) worked well.  I found the XL text was a little larger.

One  problem I had  initially was I first downloaded the files with a Windows PC and then copied them to Android...seemed easier to navigate with the PC.  For reasons and don't understand, the PC downloaded render themes resulted in blank maps again.   The directly downloaded to android render themes worked well.

Lots to learn about OSM, Mapsforge, OpenAndromaps, etc.,  hard to know where to start!  But now APRSDroid offline is working great.

Thanks again, a great app!

David KE6MOM
